main/main.c

```diff
@@ -611,6 +611,7 @@ PHP_INI_BEGIN()
 #ifdef PHP_WIN32
 	STD_PHP_INI_BOOLEAN("windows.show_crt_warning",		"0",		PHP_INI_ALL,		OnUpdateBool,			windows_show_crt_warning,			php_core_globals,	core_globals)
 #endif
+	STD_PHP_INI_ENTRY("syslog.ascii_only",			"0",		 PHP_INI_SYSTEM,	OnUpdateBool,		syslog_ascii_only,		php_core_globals, 		core_globals)
```

Comment:
This is turning into a bike-shedding exercise.

The original question was trivial: how to handle strings with embedded newlines?

Splitting into two issues and two PR's.

See [bug #75077](https://bugs.php.net/bug.php?id=75077) for the issue related to non-NVT ASCII.